ztp/source-cluster-crs/cluster-crs.yaml

```diff
@@ -84,6 +85,8 @@ spec:
     enabled: true
   searchCollector:
     enabled: false
+  workManager:
+    enabled: true
```

Comment:
I think that workManager must default to enabled (my clusters have it enabled although it is not in the applied CR), however I'm happy to have it explicitly defined here.